util/MsSqlMigratorUtility/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Bit.Migrator;
+using Microsoft.Extensions.Logging;
```

Comment:
⚠️ Lint.

Reply:
Those linting errors are strange. Without the usings the project isn't building, and with them, the linter is complaining. I ran the liter on my machine, and everything seems fine:
![image](https://user-images.githubusercontent.com/13666432/222205165-e9d35c82-367a-4014-bee0-dc27dd631c7f.png)

Reply:
Yeah, you definitely need these so I don't understand the complaint. Try changing the project file a bit and see if that prods something.